tests/mssql-tests.py

```diff
@@ -56,10 +58,12 @@ def main():
         print_usage_error(path.basename(__file__), e)
         exit(2)
     try:
-        # For our tests, tds_version 7.1 is enough
+        # For our tests, tds_version 7.1 is enough and is the default
```

Comment:
now that `tds_version` is an argument, just remove this whole block of comments. Not needed anymore :-)